test/scripts/forked_state/docker-compose.yaml

```diff
@@ -3,7 +3,7 @@ services:
     image: ghcr.io/foundry-rs/foundry:nightly-f6208d8db68f9acbe4ff8cd76958309efb61ea0b
     restart: unless-stopped
     ports: ["8545:8545"]
-    entrypoint: [ "anvil", "--fork-url", "https://gateway.tenderly.co/public/sepolia", "--host", "0.0.0.0" ]
+    entrypoint: [ "anvil", "--fork-url", "https://gateway.tenderly.co/public/sepolia", "--host", "0.0.0.0", "--block-time", "0.1", "--gas-price", "1", "--silent" ]
```

Comment:
Added this back in as it slows down the initial deployment of infrastructure and tests when removed.

Reply:
Don't you just leave the infra running? I don't bring this up/down that often